example-app/SantokuApp/src/fixtures/msw/datas/eventData.ts

```diff
@@ -4,6 +4,11 @@ import {partnerAccount} from './accountMinData';
 import {db} from '../db';
 
 export const eventData = () => {
+  const date = new Date();
+  // 日付を1年後に設定する
+  date.setFullYear(date.getFullYear() + 1);
+  const formattedDate = new Date(date.getTime() - date.getTimezoneOffset() * 60000).toISOString();
```

Comment:
> `date.getTime() - date.getTimezoneOffset() * 60000`

873c39dca7fa91231eabc305f7f0f8e74f767d3f `toISOStringで日付がずれてしまう現象を解消し、不要に修正した日付を元に戻す`

CI(GitHub Actions)とlocalで差が出てしまう対策ならば、 日時固定側でやると良さそうです

- https://github.com/ws-4020/mobile-app-crib-notes/pull/1256#discussion_r1360246452

```
  // テスト時は2023/6/1(UTC)で日付を固定する。CIとlocalで差が出ないようにISO書式で指定する
  jest.setSystemTime(new Date('2023-06-01T00:00:00+00:00'));
```

上記対応だけだとローカルのテストが落ちるので、↓を入れます

```suggestion
  const formattedDate = date.toISOString();
```

Reply:
対応commit: https://github.com/ws-4020/mobile-app-crib-notes/pull/1256/commits/bfd4f97afc0164631709a424eb053ab9a9429971